redash/templates/unsupported.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
This file should be in `client/app` and we need to make sure webpack copies it into dist.

Reply:
Done.

Just realized `index.html` and `multi_org.html` are exactly the same. Intentional?

Reply:
You still need a Flask route serving it.

Reply:
The page is configured as a static asset `https://domain.com/static/unsupported.html` - so doesn't need it, nay?

Reply:
You're right. I didn't notice you updated the URL 👍